recipe/meta.yaml

```diff
@@ -33,7 +34,7 @@ test:
 
 about:
   home: http://ab-initio.mit.edu/nlopt
-  license: LGPL 2.1
+  license: LGPL-2.1
```

Comment:
It's either `LGPL-2.1-only` or `LGPL-2.1-or-later`. You'll have to check their docs

Reply:
just saw, sorry.